types/canvasjs/index.d.ts

```diff
@@ -1,992 +1,1690 @@
-// Type definitions for CanvasJS v1.5.1
+// Type definitions for CanvasJS 1.9
 // Project: http://canvasjs.com/
-// Definitions by: Mark Overholt <https://github.com/mover5>
+// Definitions by: ShuYin Zhang  <https://github.com/brutalimp>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-declare namespace CanvasJS {
+export namespace CanvasJS {
```

Comment:
And to restore its functionality as a global, all you should need to do is remove the `export` modifier from this namespace.

If you do, somehow, have it bundled into a situation where it is usable as a module, both behaviors need to be preserved. To do so, remove the `export` modifier from this namespace, add `export = CanvasJS` and `export as namespace CanvasJS` to the file.